packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -358,6 +366,40 @@ abstract class ImplicitlyAnimatedWidgetState<T extends ImplicitlyAnimatedWidget>
   /// [forEachTween] should override [didUpdateTweens] to update those
   /// properties. Dependent properties should not be updated within
   /// [forEachTween].
+  ///
+  /// {@tool sample}
+  /// ```dart
```

Comment:
It's good practice to give a one or two sentence introduction within the `{@tool sample}` section that briefly explains in prose what the example code is accomplishing.

Reply:
Ack. Done.